QuizAppNew/View/HomeView.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 struct HomeView: View {
     @State var show = false
-    
+    @StateObject var viewModel : QuestionModel //create instance for questionViewModel
```

Comment:
@StateObject var viewModel : QuestionModel //create instance for questionViewModel
why need it.

Reply:
@State var read and write kore value manage kore .. ekahne show er value ta state property wrapper er maddhome value manage korbe..